libraries/shared/factories/event/repository.go

```diff
@@ -167,6 +168,15 @@ func PersistModels(models []InsertionModel, db *postgres.DB) error {
 	return tx.Commit()
 }
 
+func isValidValue(value interface{}) bool {
+	switch value.(type) {
+	case *pq.StringArray:
```

Comment:
Not really sure why `driver.IsValue` doesn't consider this a valid type. But `pq.StringArray` definitely works with the `tx.Exec` function; in fact I'm pretty sure that's the type's main use-case.